_opensearch/query-dsl/term.md

```diff
@@ -57,9 +71,11 @@ GET shakespeare/_search
 }
 ```
 
-We don’t get back any matches (`hits`). This is because the term “To be, or not to be” is searched literally in the inverted index, where only the analyzed values of the text fields are stored. Term-level queries aren't suited for searching on analyzed text fields because they often yield unexpected results. When working with text data, use term-level queries only for fields mapped as keyword only.
+The response indicates zero `hits` to denote a match. This is because the term “To be, or not to be” is searched literally in the inverted index, where only the analyzed values of the text fields are stored.
```

Comment:
```suggestion
The response contains no matches, indicated by zero `hits`. This is because the term “To be, or not to be” is searched literally in the inverted index, where only the analyzed values of the text fields are stored.
```